### PR TITLE
fix: remove use of polyfill.io

### PR DESCRIPTION
**Pull request by jluxenberg**
_A summary of your pull request, including the what change you're making and why._

## Testing

This change was tested as part of a general effort to remove [polyfill.io](http://polyfill.io/) from all installation methods

### CODE REVIEW

**Review by pooyaj — APPROVED** [review]

File: packages/browser-destinations/destinations/commandbar/src/init-script.ts [hunk 1]
```diff
@@ -79,8 +79,5 @@ export function initScript(orgId) {
         t(''.concat(u, '/latest/').concat(o, '?').concat(d.join('&')), !0)
     }
   }
-  void 0 === Object.assign || 'undefined' == typeof Symbol || void 0 === Symbol.for
-    ? ((a.__CommandBarBootstrap__ = r),
-      t('https://polyfill.io/v3/polyfill.min.js?version=3.101.0&callback=__CommandBarBootstrap__&features=' + n))
```
Comment (marinhero) [inline packages/browser-destinations/destinations/commandbar/src/init-script.ts:84]: Another option that you have is to use a CloudFlare version of this like they recommend on their [blog](https://blog.cloudflare.com/automatically-replacing-polyfill-io-links-with-cloudflares-mirror-for-a-safer-internet).

**Comment by marinhero** [comment]
Hey @jluxenberg please let us when this is ready to be released.

File: packages/browser-destinations/destinations/commandbar/src/init-script.ts (hunk 1 above)
Comment (marinhero) [inline packages/browser-destinations/destinations/commandbar/src/init-script.ts:84]: https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?version=3.101.0&callback=__CommandBarBootstrap__&features=

Comment (marinhero) [reply packages/browser-destinations/destinations/commandbar/src/init-script.ts:84]: Seems to work as a drop-in replacement.

Comment (jluxenberg) [reply packages/browser-destinations/destinations/commandbar/src/init-script.ts:84]: Thanks @marinhero! We've decided that these polyfills are not necessary based on the browsers we support, so it's okay to remove entirely (I'm an engineer at CommandBar).

**Comment by jluxenberg** [comment]
> Hey @jluxenberg please let us when this is ready to be released.

Ready for release, thanks for taking a look at this! ✅

**Review by marinhero — APPROVED** [review]

**Comment by marinhero** [comment]
@jluxenberg thank you! This will start to go live on Monday during our regular release cycle. Cheers!

**Comment by joe-ayoub-segment** [comment]
hi @jluxenberg and @marinhero this PR has been deployed!

**Comment by jluxenberg** [comment]
Thank you @joe-ayoub-segment for getting this deployed!

**Pull request merged**